code/modules/mob/mob_helpers.dm

```diff
@@ -71,17 +71,10 @@
 	return 0
 
 /proc/isipc(A)
+	. = 0
 	if(istype(A, /mob/living/carbon/human))
-		switch(A:get_species())
-			if ("Baseline Frame")
-				return 1
-			if ("Industrial Frame")
-				return 1
-			if ("Shell Frame")
-				return 1
-			if ("Hunter-Killer")
-				return 1
-	return 0
+		var/mob/living/carbon/human/H = A

```

Comment:
I wonder what the cost difference between this var instantiation and initializing `A` as `/mob/living/carbon/human/A` is. Basically,

```
/proc/isipc(/mob/living/carbon/human/H)
	. = FALSE
	if (istype(H))
		. = !!global.machine_species[H.get_species()]
```

Could be one less operation?